libs/ui/src/progress_ellipsis.test.tsx

```diff
@@ -5,11 +5,5 @@ import { ProgressEllipsis } from './progress_ellipsis';
 
 test('renders ProgressEllipsis', () => {
   const { container } = render(<ProgressEllipsis />);
-  expect(container).toMatchInlineSnapshot(`
-    <div>
-      <span
-        class="sc-bdvvaa etEPtU"
-      />
-    </div>
-  `);
+  expect(container).toMatchSnapshot();
```

Comment:
This should be removed & replaced by a more specific test. (not necessarily for this PR)